### DISCUSSION TITLE
Is production ready with pgbouncer + postgres?

### DISCUSSION THREAD

**Discussion by budibong** [sqlc-dev/sqlc#1906] [Q&A]
Hi everyone,

I am new to golang ecosystem and i am thinking to adopt sqlc as new tech stack for my team. 

Does anyone have successfully using sqlc for production application with high volume of requests? Especially using pgbouncer + postgres as db.

Any answer or advice will be very appreciated. Thanks

**Comment by kyleconroy** [comment]
For PostgreSQL, sqlc can either generate code that uses [database/sql](https://pkg.go.dev/database/sql) and [lib/pq](https://github.com/lib/pq) or [jackc/pgx](https://github.com/jackc/pgx). Since pgbouncer is wire-compatible with PostgreSQL, I believe both options should work without modifications.
